4-forms-validations/form.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="author" content="augusticor" />
+        <link rel="stylesheet" href="formStyles.css">
```

Comment:
keep your file names in lower case and if they have multiple words, use dashes. `form-styles.css`